src/Phpunit/DatabaseTestListener.php

```diff
@@ -110,39 +110,44 @@ private function setUpPhpcrDatabase($suite)
     {
         echo PHP_EOL.PHP_EOL;
 
+        // initialize PHPCR DBAL (new way)
         $process = $this->processBuilder
             ->setArguments(array_merge($this->prefix, array('doctrine:phpcr:init:dbal', '--drop', '--force')))
             ->getProcess();
+
         $process->run();
 
-        while (true) {
-            if ($process->isTerminated()) {
```

Comment:
all these while loops and tests were not needed, as `->run()` will always wait until the command is terminated (I probably didn't know this the first time I used the Process component)